File: 1-Iniciante/1040-Media-3/1040.py
n = input().split()
media = round((float(n[0]) * 2 + float(n[1]) * 3 + float(n[2]) * 4 + float(n[3])) / 10, 1)
print ("Media:",media)
if media >= 7:
    print ("Aluno aprovado.")
elif media < 5:
    print ("Aluno reprovado.")
elif media >= 5 and media <= 6.9:
    print ("Aluno em exame.")
    e = float(input())
    print("Nota do exame:", e)
    media = (media + e) / 2
    if media > 5:
        print ("Aluno aprovado.")
    else:
        print ("Aluno reprovado.")
    print ("Media final:", media)

File: 1-Iniciante/1040-Media-3/1040.cpp
#include <cstdio>

int main() {
    float media = 0, input;
    for (int i = 0; i < 3; ++i) {
        scanf("%f", &input);
        media += input * (i+2);
    }
    scanf("%f", &input);
    media += input;
    media /= 10;
    printf ("Media: %.1f\n", media);
    if (media >= 5 && media <= 6.9) {
        printf("Aluno em exame.\n");
        scanf("%f", &input);
        printf ("Nota do exame: %.1f\n", input);
        media = (media + input) / 2;
        if (media >= 5) {
            printf ("Aluno aprovado.\n");
        }
        else {
            printf ("Aluno reprovado.\n");
        }
        printf ("Media final: %.1f\n", media);
    }
    else if (media >= 7) {
        printf ("Aluno aprovado.\n");
    }
    else {
        printf ("Aluno reprovado.\n");
    }
    return 0;
}